.github/workflows/scripts/install-build-deps.sh

```diff
@@ -22,6 +22,22 @@ do
   fi
 done
 
+if [ -v INSTALL_COMPUTE_RUNTIME ]; then
+    echo "install Compute Runtime"
+    wget -U "$USER_AGENT" --retry-connrefused --waitretry=5 --read-timeout=20 --timeout=15 -t 5 \
+        https://github.com/intel/intel-graphics-compiler/releases/download/igc-1.0.14828.8/intel-igc-core_1.0.14828.8_amd64.deb
```

Comment:
I'd prefer not to keep/update CR version in two more files in addition to what we already have in Dockerfiles unless we really need to use a particular version of CR runtime here. Could you try to install here version-independent agama packages like we do in https://github.com/ispc/ispc/blob/main/docker/ubuntu/20.04/xpu_ispc_build/Dockerfile#L46?
Docs: https://dgpu-docs.intel.com/driver/installation.html

Reply:
Fixed